### PR TITLE
blog: Running Flotta on Raspberry Pi

### PR DESCRIPTION
**Pull request by jakub-dzon**
This blog post depends on https://github.com/project-flotta/flotta-operator/pull/193.

### CODE REVIEW

File: _posts/2022-04-15-flotta-and-raspberry-pi.markdown [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+$                           
+```
+
+On the computer with $KUBECONFIG pointing to our cluster, let's forward required port to make HTTP API public:
+```shell
+$ kubectl port-forward service/flotta-operator-controller-manager -n flotta 8043 --address 0.0.0.0
+Forwarding from 0.0.0.0:8043 -> 8043
+```
+
+IP address of the machine exposing the HTTP API is `10.220.200.106`.
```
[the remark this answers is not in the document]
Comment (jakub-dzon) [reply _posts/2022-04-15-flotta-and-raspberry-pi.markdown]: Done.

**Review by masayag — APPROVED** [review]
Very nice!

**Review by pkliczewski — APPROVED** [review]

File: _posts/2022-04-15-flotta-and-raspberry-pi.markdown [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+On the device, let's execute the uploaded script, providing IP address of machine making Flotta Operator HTTP API public:
+```shell
+./install-agent-rpm-ostree.sh -i 10.220.200.106
+```
+At the end of its execution, the script will restart the Raspberry Pi device; it may take a while for the device to 
+be available again. 
+
+Let's monitor `EdgeDevice` object in our cluster to see when the device becomes available:
```
[the remark this answers is not in the document]
Comment (eloycoto) [reply _posts/2022-04-15-flotta-and-raspberry-pi.markdown:108]: :+1:

**Review by eloycoto — COMMENTED** [review]
Overall LGTM, but I'll wait until device approval to merge  to update to the right path.

**Comment by jakub-dzon** [comment]
@eloycoto - how will that change the contents of this post?

**Review by machacekondra — APPROVED** [review]

**Comment by eloycoto** [comment]
At the ends, nothing.. if auto-approve is enabled, so can be merged.

**Pull request merged**